api/v1/prefectworkpool_types.go

```diff
@@ -55,36 +54,6 @@ type PrefectWorkPoolSpec struct {
 	DeploymentLabels map[string]string `json:"deploymentLabels,omitempty"`
 }
 
-type PrefectServerReference struct {
```

Comment:
This was hoisted up to be reusable between `PrefectWorkPool` and `PrefectDeployment`